lib/lotus/commands/generate/migration.rb

```diff
@@ -0,0 +1,37 @@
+require 'lotus/utils/string'
+require 'lotus/generators/migration'
+
+module Lotus
+  module Commands
+    module Generate
+      class Migration 
+        
+        attr_reader :migration_name, :migration_time, :migration_class, :cli, :source, :target
+
+        def initialize(name, environment, cli)
+          name = Lotus::Utils::String.new(name)
+          @migration_name = name.underscore
```

Comment:
What do you think about `Tell, Don't Ask`? I think it'd better to have a private method `generate_name` here.